docs/source/CartesianConfig.rst

```diff
@@ -4,11 +4,16 @@
 Cartesian Configuration
 =======================
 
-Cartesian Configuration is a highly specialized way of providing lists
-of key/value pairs within combination's of various categories. The
-format simplifies and condenses highly complex multidimensional arrays
-of test parameters into a flat list. The combinatorial result can be
-filtered and adjusted prior to testing, with filters, dependencies, and
+In software testing, key/value pairs are usually used as test input parameters.
+
+Cartesian product is a mathematical operation which creates a set from
+multiple sets. Given sets A and B, the cartesian product of both is
+the set of all ordered pairs (a,b) where a belongs to A and b belongs to B.
+
+In Avocado VT, cartesian configuration is a method of defining key/value pairs
+which are later combined using the cartesian product concept. Each element of
+this product is used as input by a different test execution. The product can
+be filtered and adjusted prior to testing, with filters, dependencies, and
```

Comment:
I easier understood the original definition, but it's probably because I'm used to that. From reviews I see @alanoe, @clebergnu and @lmr preferring this one and as I can understand it as well, let's go with this one...